.github/build.sh

```diff
@@ -1,3 +1,28 @@
 #!/bin/sh
 curl -fsLO https://raw.githubusercontent.com/scijava/scijava-scripts/master/ci-build.sh
 sh ci-build.sh
+
+# Record the last time of cache modification before running melting-pot
+cache_modified_pre=0
+cache_modified_post=0
+
+if [ -f "~/.cache/scijava/melting-pot" ]; then
+  cache_modified_pre=$(find ~/.cache/scijava/melting-pot -printf '%T@\n' | sort -r | head -n 1)
+  cache_modified_pre=${cache_modified_pre%.*}
+fi
+
+# run melting-pot
+sh tests/run.sh
```

Comment:
Remove `sh`, just use `tests/run.sh` directly; it's executable.

Reply:
Secondly, if this returns non-zero, we need the script to die here. You can either:
```
tests/run.sh || exit $?
```
(I *think* this will be the exit code of `run.sh` then...)

OR, you can put `set -e` at the top of the `.github/build.sh` script.

Let's try `set -e` at the top first, because we also need `ci-build.sh` to die if it fails...

Reply:
Oh, for completeness: one other solution to this issue is to put `&&` suffix at the end of every line of the script, thus chaining all the commands together, such that once any of them fails, the subsequent lines do not execute and the script terminates with the failing exit code.